example/package.json

```diff
@@ -11,14 +11,14 @@
     "react-dom": "^15.4.1"
   },
   "devDependencies": {
-    "ca-webpack-config": "^1.0.0",
     "babel-eslint": "^7.1.1",
     "eslint-config-ca": "^1.0.0",
-    "webpack": "*",
-    "webpack-dev-server": "*"
+    "webpack": "^2.2.1",
+    "webpack-dev-server": "^2.4.1"
   },
   "scripts": {
     "build": "webpack -d",
+    "prestart": "cd ../ && npm run build",
     "start": "webpack-dev-server -d"
```

Comment:
@alebiavati `html-webpack-plugin` isn't run by only running `webpack-dev-server`. I'm not sure what the best way around this is?